templates/credentials/schedule.html

```diff
@@ -110,19 +110,22 @@ <h1 class="p-heading--2">
     }
 
     const setError = () => {
-    if (maintenance_error === 'True') {
-      const elem = document.getElementById("error-message");
-      const startFormatted = formatDateTime(maintenance_start, navigator.language);
-      const endFormatted = formatDateTime(maintenance_end, navigator.language);
-      if (elem) {
-        elem.innerHTML = `Scheduled time should be outside of the maintenance window. Please schedule the exam before <strong>${startFormatted}</strong> or after <strong>${endFormatted}</strong>`;
+      if (maintenance_error === 'True') {
```

Comment:
I would suggest doing this inside the html using jinja conditions. This way we don't write HTML in JS